src/components/alert-banner/alert-banner.js

```diff
@@ -44,30 +42,27 @@ export default function() {
   */
   function addAlertCookie(alert) {
     const cookieName = cookieBuilder.dataset(alert, 'cookie');
-    if (cookieName) {
+    if (cookieName)
       cookieBuilder.createCookie(
           cookieName,
           'dismissed',
           cookieBuilder.getDomain(window.location, false),
           360
       );
-    }
   }
 
   const alerts = document.querySelectorAll('.js-alert');
 
-  if (alerts.length) {
+  if (alerts.length)
     for (let i=0; i <= alert.length; i++) {
       if (!checkAlertCookie(alerts[i])) {
         const alertButton = document.getElementById('alert-button');
         displayAlert(alerts[i]);
-        alertButton.addEventListener('click', e => {
+        alertButton.addEventListener('click', (e) => {
             alerts[i].classList.add('hidden');
             addAlertCookie(alerts[i]);
           });
-      } else {
+      } else
         alerts[i].classList.add('hidden');
-      }
     }
-  }
 }
```

Comment:
There was one eslint error I couldn't get around. I got `Unnecessary { after 'for' condition` but when I removed the `{}` from the for loop it will break the loop. I tried multiple things without success. :(

Reply:
I'm not able to replicate the error you're getting for this.